hydra-web/src/templates/host.selectable.html

```diff
@@ -19,15 +19,15 @@
     <button id="hostRebalanceButton" class='btn btn-default btn-blue btn-hide-zero'>
         Rebalance
     </button>
-    <button id="hostFailFsOkayButton" class='btn btn-default btn-red btn-hide-zero'>
-        Fail (filesystem okay)
-    </button>
-    <button id="hostFailFsDeadButton" class='btn btn-default btn-red btn-hide-zero'>
-        Fail (filesystem dead)
+    <button id="hostFailFsOkayButton" class='btn btn-default btn-blue btn-hide-zero'>
+        Queue to Fail
     </button>
     <button id="hostFailCancelButton" class='btn btn-default btn-blue btn-hide-zero'>
         Cancel failure
     </button>
+    <button id="hostFailFsDeadButton" class='btn btn-default btn-red btn-hide-zero btn-show-one'>
```

Comment:
I would recommend against leaving btn-hide-zero on this button, because that implies certain behavior, and it's not obvious what the business logic is. Instead, handle the two types of buttons separately in the javascript.